app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [OPTIONAL] I will recommend that you enable `:confirmable`. The `:confirmable` module in the Rails Devise gem provides an email confirmation step to the user registration process. After registering, users receive an email with a unique confirmation link. By clicking the link, users confirm their email address and activate their account. This feature helps verify the legitimacy of user accounts and prevents fake or unauthorized registrations.